saviSimulation/savi/jason_processing/SAVIWorld_model.java

```diff
@@ -162,7 +162,7 @@ public void setup() {
 			if(RANDOM_SEED != -1) {
 				rand = new Random(RANDOM_SEED+i);
 			}	
-				UaV uav = new UaV(i, new PVector(rand.nextInt(X_PIXELS) + 1, rand.nextInt(Y_PIXELS) + 1, UAV_SIZE/2), UAV_SIZE,"demo", 
+				UaV uav = new UaV(i, new PVector(rand.nextInt(X_PIXELS) + 1, rand.nextInt(Y_PIXELS) + 1, UAV_SIZE/2), UAV_SIZE,"uav",
```

Comment:
At a glance (at run time) it looks like the UAVs are doing similar things to the UGVs. I'm going to take a closer look a the code to understand the behaviour, but it may be worth double checking that the correct asl files are in fact being loaded properly.